cocos/spine/lib/spine-define.ts

```diff
@@ -42,6 +43,50 @@ function overrideDefineArrayProp (prototype: any, getPropVector: any, name: stri
     });
 }
 
+function overrideDefineArrayArrayProp (prototype: any, getPropVector: any, name: string): void {
+    Object.defineProperty(prototype, name, {
+        get (): any[] {
+            const array: any[] = [];
+            const vectors = getPropVector.call(this);
+            const count = vectors.size();
+            for (let i = 0; i < count; i++) {
+                const vectorI = vectors.get(i);
+                const countJ = vectorI.size();
+                const arrayJ: any[] = [];
+                for (let j = 0; j < countJ; j++) {
+                    arrayJ.push(vectorI.get(j));
+                }
+                array.push(arrayJ);
+            }
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+            return array;
+        },
+    });
+}
+
+function overrideDefineArrayPropGetSet (prototype: any, getPropVector: any, setPropVector: any, Type: any, name: string): void {
+    Object.defineProperty(prototype, name, {
+        get (): any[] {
+            const array: any[] = [];
+            const vectors = getPropVector.call(this);
+            const count = vectors.size();
+            for (let i = 0; i < count; i++) {
+                const objPtr = vectors.get(i);
+                array.push(objPtr);
+            }
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+            return array;
+        },
+        set (value: any[]) {
+            const vectors = new Type();//spine.wasmUtil.wasm.VectorFloat();
+            for (const obj of value) {
```

Comment:
Is `vectors.reserve()` exported to JS ? If so, it's better to call it after `new Type()`.
```js
const vectors = new Types();
vectors.reserve(value.length);
```